### PR TITLE
Use xml streams in `XmlPrimaryChecksums`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #440 
Used xml streams in `XmlPrimaryChecksums` and made this class return map of the packages names and checksum. Changed the field from path to input stream.

### CODE REVIEW

File: src/main/java/com/artipie/rpm/meta/XmlPrimaryChecksums.java [hunk 1]
```diff
@@ -4,53 +4,86 @@
...
         }
         return res;
     }
+
+    /**
+     * Checks event.
+     * @param event Event
+     * @param tag Xml tag
+     * @return True is this event is given xml tag
+     */
```
Comment (genryxy) [inline src/main/java/com/artipie/rpm/meta/XmlPrimaryChecksums.java:84]: this comment is a bit confused

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — DISMISSED** [review]
@olenagerasimova thanks, looks good

**Comment by olenagerasimova** [comment]
@genryxy please, approve again, I've fixed windows tests

**Review by genryxy — APPROVED** [review]

**Pull request merged**